docs/src/markdown-pages/aptitudes/system.operating-system.md

```diff
@@ -0,0 +1,11 @@
+---
+name: "Operating System"
+links_js: "operatingSystem"
+---
+Returns a string representation of the host Operating System.
+
+Possible values:
```

Comment:
Doesn't OH support Linux still?

Reply:
Good question, I wasn't sure. But based on the [Installation Guide](https://oliveai.dev/guides/olive-helps-install-guide) on the Dev Hub, we do not support Linux, only macOS and Windows.